readme.txt

```diff
@@ -79,6 +79,9 @@ The source code is freely available [in GitHub](https://github.com/Automattic/wo
 
 == Changelog ==
 
+= 2.5.5 - 2024-xx-xx =
+* Add - Don't register functionality if Woo Shipping or Woo Tax is active.
```

Comment:
Suggestion (not blocking): from a merchant/developer PoV, then we could perhaps tease the the future deprecation a bit more and clarify what even "Woo Shipping" and "Woo Tax" is when they read this changelog; otherwise they might just be confused about the plugins being shipping and tax functionality in WCS&T?

```suggestion
* Add - Prevent upcoming Woo Shipping and Woo Tax plugins from running in parallel with this plugin unless both are active, then they will take over for this plugin.
```

**Duplicate of other comment**